docs/tools-reference/tools/how-to-install-midnight-commander.md

```diff
@@ -177,4 +180,4 @@ The first thing to do is install the utility.
 
 ## Conclusion
 
-Midnight Commander has many more features than those presented here. Before making a lot of customizations to the default settings, it's a good idea to backup your settings by copying `~/.config/mc/` to your local computer. You can also clone these settings to other servers by copying them to the same location.
+Midnight Commander has many more features than those presented here. Before making many customizations to the default settings, it's a good idea to backup your settings by copying `~/.config/mc/` to your local computer. You can also clone these settings to other servers by copying them to the same location.
```

Comment:
"Before" should be "after" here, the idea being the user wouldn't want to spend another XY minutes to make the same customizations in the future.